package.json

```diff
@@ -36,11 +39,14 @@
   },
   "homepage": "https://github.com/mapbox/assembly#readme",
   "dependencies": {
+    "@mapbox/batfish": "^0.5.0",
     "autoprefixer": "^6.6.1",
     "concat-with-sourcemaps": "^1.0.4",
     "cp-file": "^4.1.1",
     "csso": "^3.1.1",
+    "es6-promise": "^4.1.1",
```

Comment:
@davidtheclark Actually the build is still failing for me when I remove this from `package.json`... do you know what might be awry? Here's what I've been doing:
1. Manually remove `es6-promise` from `package.json`
2. `rm -rf node_modules`
3. `rm yarn.lock`
4. `yarn install`

That should work, right? When I do that, the build fails with ` Module not found: Error: Can't resolve 'es6-promise/auto'`